<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>722</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mesh Fitting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>100</number>
           </attribute>
           <column>
            <property name="text">
             <string>Visible</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QToolBox" name="toolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_fitting">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>398</width>
              <height>531</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Segmentation</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>ASM Configuration</string>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>PCs to Fit:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="pcsToFitSpinBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Mahalanobis Weight:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="mWeightDblSpinBox"/>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Model Discretisation*:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Profile Extent*:</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QDoubleSpinBox" name="pExtDblSpinBox1">
                    <property name="minimum">
                     <double>-1000.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="pExtDblSpinBox2"/>
                  </item>
                 </layout>
                </item>
                <item row="8" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Profile Model File:</string>
                  </property>
                 </widget>
                </item>
                <item row="8" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLineEdit" name="profileModelLineEdit"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="profileModelButton">
                    <property name="text">
                     <string>...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="9" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Search Distance:</string>
                  </property>
                 </widget>
                </item>
                <item row="9" column="1">
                 <widget class="QSpinBox" name="searchDistSpinBox"/>
                </item>
                <item row="10" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Max Iterations:</string>
                  </property>
                 </widget>
                </item>
                <item row="10" column="1">
                 <widget class="QSpinBox" name="maxItSpinBox"/>
                </item>
                <item row="4" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QSpinBox" name="surfDiscSpinBox1"/>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="surfDiscSpinBox2"/>
                  </item>
                 </layout>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Profile Samples*:</string>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QSpinBox" name="profileSamplesSpinBox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>*Must match the profile model</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="fitButtonsGroup">
               <item row="1" column="1">
                <widget class="QPushButton" name="acceptButton">
                 <property name="text">
                  <string>Accept</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="resetButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="abortButton">
                 <property name="text">
                  <string>Abort</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="segButton">
                 <property name="text">
                  <string>Segment</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="errorGroup">
               <property name="title">
                <string>Segmentation Results</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="RMSELabel">
                  <property name="text">
                   <string>RMSE:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="RMSELineEdit">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="pFracLabel">
                  <property name="whatsThis">
                   <string>Percentage of landmarks that have converged to their texture match.</string>
                  </property>
                  <property name="text">
                   <string>Convergence %:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="pFracLineEdit">
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Screenshot">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>191</width>
              <height>143</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Screenshots</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="pixelsXLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Pixels X:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="screenshotPixelXLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>800</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="pixelsYLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Pixels Y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="screenshotPixelYLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>600</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="screenshotFilenameLabel">
               <property name="text">
                <string>Filename:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="screenshotFilenameLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>screenshot.png</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="screenshotSaveButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Save Screenshot</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="MayaviSceneWidget" name="MayaviScene" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MayaviSceneWidget</class>
   <extends>QWidget</extends>
   <header>gias2/mappluginutils/mayaviviewer/mayaviscenewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>pcsToFitSpinBox</tabstop>
  <tabstop>mWeightDblSpinBox</tabstop>
  <tabstop>surfDiscSpinBox1</tabstop>
  <tabstop>surfDiscSpinBox2</tabstop>
  <tabstop>pExtDblSpinBox1</tabstop>
  <tabstop>pExtDblSpinBox2</tabstop>
  <tabstop>profileSamplesSpinBox</tabstop>
  <tabstop>profileModelLineEdit</tabstop>
  <tabstop>profileModelButton</tabstop>
  <tabstop>searchDistSpinBox</tabstop>
  <tabstop>maxItSpinBox</tabstop>
  <tabstop>segButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>abortButton</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>RMSELineEdit</tabstop>
  <tabstop>pFracLineEdit</tabstop>
  <tabstop>screenshotPixelXLineEdit</tabstop>
  <tabstop>screenshotPixelYLineEdit</tabstop>
  <tabstop>screenshotFilenameLineEdit</tabstop>
  <tabstop>screenshotSaveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
